webpack/_config-builder.js

```diff
@@ -30,26 +29,28 @@ const baseRules = [
       cacheDirectory: true,
     },
   },
-  { test: /\.(txt|yaml)$/, loader: "raw-loader" },
+  { test: /\.(txt|yaml)$/,
+    type: "asset/source", 
+  },
   {
     test: /\.(png|jpg|jpeg|gif|svg)$/,
-    use: [
-      {
-        loader: "url-loader",
-        options: {
-          esModule: false,
-        },
-      },
-    ],
+    type: "asset/inline",
   },
   {
     test: /\.(woff|woff2)$/,
-    loader: "url-loader?",
-    options: {
-      limit: 10000,
-    },
+    type: "asset",
+  },
+  { test: /\.(ttf|eot)$/,
+    type: "asset/resource",
+  },
+  {
```

Comment:
When I remove this loader, it has no effect to the build system. Can you please verify?

Reply:
This is true. Probably a legacy configuration to load fonts, but since we don't load these types of font anymore, will remove this configuration setting.